app/controllers/api/accounts_controller.rb

```diff
@@ -40,13 +36,17 @@ def update
     account.attributes = model_params
     account.encrypt(decrypted_team_password(team))
     account_move_handler.move if account.folder_id_changed?
-    account.save!
-    render_json account
+    if account.save!
```

Comment:
if you use the "bang" operator here, it raises an error if any validation's failing ....

add test case ?